steps/urls-scanner/1.0.6/step.yml

```diff
@@ -0,0 +1,53 @@
+title: Mobile apps URLS scanner
+summary: |
+  Scan source code to extract all http/https urls and execute an analyze of those urls with SSLLabs
+description: |
+  Scan source code to extract all http/https urls and execute an analyze of those urls with SSLLabs
+website: https://github.com/odemolliens/bitrise-step-urls-scanner
+source_code_url: https://github.com/odemolliens/bitrise-step-urls-scanner
+support_url: https://github.com/odemolliens/bitrise-step-urls-scanner/issues
+published_at: 2021-03-26T10:14:27.263955+01:00
+source:
+  git: https://github.com/odemolliens/bitrise-step-urls-scanner.git
+  commit: dae66c10d568fd3b77204257427b928c3d99c690
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+- flutter
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: ssllabs-scan
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- opts:
+    description: |
+      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
+      The list set by default can help you to remove urls that are only used for development perspective.
+    is_expand: true
+    is_required: false
+    summary: Analyze HTTPS URLs with SSLLabs and generate report for each domain
+    title: Analyze HTTPS URLs with SSLLabs
+    value_options:
+    - "yes"
+    - "no"
+  ssl_labs_scan: "no"
+- black_list: apache.org;cloudflare.com;bintray.com;w3.org;apple-mapkit.com;apple.com;mzstatic.com;mozilla.org;github.io;github.com;youtube.com;withgoogle.com;android.com;google.com;jitpack.io;microsoft.com;firebaseio.com
+  opts:
+    description: |
+      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
+      The list set by default can help you to remove urls that are only used for development perspective.
```

Comment:
```suggestion
      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
      The list set by default can help you to remove URLs that are only used for a development perspective.
```

Reply:
Thanks a lot, @sylank for the feedback
I applied your changes here #2981